### PR TITLE
Doc: SNS Overview

### PR DESCRIPTION
**Pull request by VenkyOS**
Uploaded the SNS overview document.

### CODE REVIEW

File: doc/SNS_Overview.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+======================
+SNS Overview
+======================
+
+This document summarizes SNS and local RAID discussions in the C2 architecture team. The goal is to provide a starting point for design level discussions and resource estimation. Note that this documents is not a substitute for a list of requirements or use-cases. Intended audience includes T1 team and C2 architects.
```
Comment (huanghua78) [inline doc/SNS_Overview.rst:5]: Replace "C2" with Motr.

Replace "Intended audience includes T1 team and C2 architects." with 
```
The intended audience of this document consists of Motr customers, architects, designers and developers
```

File: doc/SNS_Overview.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- Design of layout formulae is a critical condition of efficient striping implementation. The class of formulae must conform to the following requirements: 
+
+  - all standard RAID patterns must be supported compactly; 
+
+  - parity declustering must (see below) be supported; 
+
+  - formulae should work efficiently with small FLDB (Fid Location Data-Base) and CLDB (Container Location Data-Base), in the face of object migration. 
+  
+- Concurrency control. Concurrent conflicting operations on a striped file must preserve consistency. Striping with redundancy introduces dependencies between data updates: parity and corresponding data must be updated atomically. T1 does not provide concurrent access to exported block devices, but still have some concurrency, e.g., during recovery.
```
Comment (huanghua78) [inline doc/SNS_Overview.rst:33]: Replace "T1" with "Current version".

Do the same for other places.

File: doc/SNS_Overview.rst (hunk 1 above)
Comment (VenkyOS) [reply doc/SNS_Overview.rst]: Incorporated.

File: doc/SNS_Overview.rst (hunk 2 above)
Comment (VenkyOS) [reply doc/SNS_Overview.rst]: Incorporated.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Review by huanghua78 — APPROVED** [review]

**Pull request merged**